sound/soc/sof/ipc4-control.c

```diff
@@ -562,6 +619,11 @@ static int sof_ipc4_widget_kcontrol_setup(struct snd_sof_dev *sdev, struct snd_s
 				ret = sof_ipc4_set_get_bytes_data(sdev, scontrol,
 								  true, false);
 				break;
+			case SND_SOC_TPLG_CTL_ENUM:
+			case SND_SOC_TPLG_CTL_ENUM_VALUE:
+				ret = sof_ipc4_set_generic_control_data(sdev, swidget,
+									scontrol, false);
+				break;
```

Comment:
we have a default value of 0dB for volumes, but for switch and enums what's the default?

Reply:
0 is the default for both switch and enum.
Switch: Off by default
Enum: item 0 (first enum entry) is selected